TFT/src/User/API/AddonHardware.c

```diff
@@ -86,20 +86,13 @@ void FIL_SFS_SetAlive(bool alive)
 bool FIL_NormalRunoutDetect(void)
 {
   static bool runout = false;
-  static uint32_t highCount = 0;
-  static uint32_t lowCount = 0;
+  static int32_t trig_balance = 0;
```

Comment:
if possible, it should be better trigBalance (same naming convention used for the other variables; e.g. pinState etc...)

Reply:
Sure, thanks for the suggestion.